langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,34 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.*;
+
+class MongoDbEmbeddingStoreIT extends EmbeddingStoreIT {
```

Comment:
```suggestion
class MongoDbEmbeddingStoreIT extends EmbeddingStoreWithFilteringIT {
```

Please extend from `EmbeddingStoreWithFilteringIT` to test filtering